charts/kafka-operator/values.yaml

```diff
@@ -12,7 +12,7 @@ operator:
   annotations: {}
   image:
     repository: ghcr.io/banzaicloud/kafka-operator
-    tag: v0.14.0
+    tag: v0.15.0
```

Comment:
Due to we defined the appVersion in `Charts.yaml` it can be used in templates as well.
Instead of using image: `"{{ .Values.operator.image.repository }}:{{ .Values.operator.image.tag }}"` in this [line](https://github.com/banzaicloud/kafka-operator/blob/361c8cd2a0991f0e94246acca4827414a4399889/charts/kafka-operator/templates/operator-deployment-with-webhook.yaml#L159) we would use `image: "{{ .Values.operator.image.repository }}:{{ .Values.operator.image.tag | default .Chart.AppVersion }}"` and in the values.yaml we would set tag to empty string. `tag: ""`